packages/workbox-routing/src/registerRoute.ts

```diff
@@ -38,11 +40,14 @@ import './_version.mjs';
  *
  * @alias workbox.routing.registerRoute
  */
-export const registerRoute = (capture, handler, method = 'GET') => {
+export const registerRoute = (
+    capture: RegExp | string | matchCallback | Route,
+    handler: handlerCallback,
```

Comment:
Both `handler` and `method` are optional the first parameter is a `Route`.

(This method is terribly overloaded.)

Reply:
Gotcha. Fixed.